test/edge_elevation.cc

```diff
@@ -1,122 +0,0 @@
-#include "test.h"
```

Comment:
we should reenable this test. the tiles support this datastrcuture and while this isnt the most extensive test we should reenable as long as we support this datastructure in the tiles